src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link is-active"
+            >
+              Apple
```

Comment:
It is a good practice to make the current page's navigation link non-interactive. Since this is the 'Apple' page, the 'Apple' link should not be clickable or should have an `aria-current="page"` attribute to indicate that it is the current page.